wiki/en/Include-Client-Commands.md

```diff
@@ -2,5 +2,5 @@
 - `--mutemyown`  Mute me in my personal mix ("headless" mode only)                                                      
 -  `-c` or `--connect`  Connect to given server address on startup, format `address[:port]`  
 -  `-j` or `--nojackconnect`  Disable auto JACK connections  
--  `--ctrlmidich`  MIDI controller channel to listen on, control number offset and consecutive CC numbers (channels). Format: `channel[;f*][;p*][;s*][;m*]` See [Tips & Tricks](Tips-Tricks-More#using-ctrlmidich-for-midi-controllers) 
+-  `--ctrlmidich`  MIDI controller channel to listen on, control number offset and consecutive CC numbers (channels) and Mute Myself CC number. Format: `channel[;f*][;p*][;s*][;m*][;o]` See [Tips & Tricks](Tips-Tricks-More#using-ctrlmidich-for-midi-controllers) 
```

Comment:
Not sure if the 'o' parameter should be followed by an asterisk. Can't remember if it's a wildcard or that the parameter is literally followed by an asterisk.

Reply:
No, the `*` means "times" (as in multiplication).  So `*8` means the control is used 8 times.

Reply:
There's also an "old style" format that I don't remember...

Reply:
Ok, thanks.

Reply:
And, just as a note, I think the order of the `f`, `p`, `s`, `m` and `o` doesn't matter... I've not checked that, though.

Reply:
No, it doesn't. Should we mention that?

Reply:
> There's also an "old style" format that I don't remember... So long as the documentation doesn't _clash_ with that, it's okay not to mention it...

I Know I originally got that format from _somewhere_ - but just can't remember any more.

Reply:
It is documented in the soundbase.cpp file:

   // parse the server info string according to definition: there is
    // the legacy definition with just one or two numbers that only
    // provides a definition for the controller offset of the level
    // controllers (default 70 for the sake of Behringer X-Touch)
    // **[MIDI channel];[offset for level]**
 
Just in case you want to add it.

Reply:
I would leave the client-commands doc as it is - no need to clutter it when there is a link to a more detailed explanation. 

My initial doubt was about _how_ the command is presented in client-commands.md. I know what the command should look like when actually used, but there's obviously a reason why it's shown as e.g. `[;f*]` and not `[;f(offset)*(cons.ch.)]` or whatever. I just couldn't remember whether the `*` represented the multiplier or was a wildcard for everything after 'f' - in which case the new parameter, 'o', should have been followed by an `*`.